src/test/java/com/google/api/codegen/testdata/library_config.baseline

```diff
@@ -19,6 +19,8 @@ interfaces:
   collections:
   - name_pattern: archives/{archive_path=**}/books/{book}
     entity_name: book
+  - name_pattern: archives/{archive}/books/{book_path=**}
```

Comment:
the name_pattern for the book entity above needs to be fixed so that the ** isn't in the middle of the pattern - maybe file an issue.